### ISSUE TITLE
ERROR! the role 'ansible-role-nginx' was not found in

### ISSUE THREAD

**Issue by Am1rr3zA** [nginxinc/ansible-role-nginx#35]
I have follow this https://www.nginx.com/blog/announcing-unified-ansible-role-nginx-nginx-plus/ to install ngnix on my EC2 instances, but when I want to create my AMI I am getiing this error

this is my yaml file:
```
- name: "Installs Gatling and other required dependencies for load testing"
  hosts: all
  remote_user: admin
  become: true
  roles:
    - role: ansible-role-nginx
    - role: common
    - role: datadog
      datadog_agent5: true # 6.x currently does not support graphite
      datadog_config:
        graphite_listen_port: 17124
        log_level: INFO
    - role: gatling
```

```
amazon-ebs: ERROR! the role 'ansible-role-nginx' was not found in /Users/aaa/xxx/projects/yyy/ansible/roles:/Users/aaa/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/Users/aaa/xxx/projects/yyy/ansible
    amazon-ebs:
    amazon-ebs: The error appears to have been in '/Users/aaa/xxx/projects/yyy/ansible/http-load-testing.yml': line 6, column 7, but may
    amazon-ebs: be elsewhere in the file depending on the exact syntax problem.
    amazon-ebs:
    amazon-ebs: The offending line appears to be:
    amazon-ebs:
    amazon-ebs:   roles:
    amazon-ebs:     - role: ansible-role-nginx
    amazon-ebs:       ^ here

```

and if I change `ansible-role-nginx` to `nginxinc.nginx ` which gice me this yaml file

```
- name: "Installs Gatling and other required dependencies for load testing"
  hosts: all
  remote_user: admin
  become: true
  roles:
    - role: nginxinc.nginx 
    - role: common
    - role: datadog
      datadog_agent5: true # 6.x currently does not support graphite
      datadog_config:
        graphite_listen_port: 17124
        log_level: INFO
    - role: gatling
```

first, it asks me to install ` nickdtodd.system-limits` with ansible-galaxy (do we really need that? of yes we need to mention in documentation) and after that, I am getting this error

```
amazon-ebs: fatal: [default]: FAILED! => {"changed": false, "cmd": "/usr/bin/apt-key adv --keyserver ha.pool.sks-keyservers.net --recv XxXxX", "msg": "Error fetching key XxXxX from keyserver: ha.pool.sks-keyservers.net", "rc": 2, "stderr": "Warning: apt-key output should not be parsed (stdout is not a terminal)\ngpg: failed to start the dirmngr '/usr/bin/dirmngr': No such file or directory\ngpg: connecting dirmngr at '/tmp/apt-key-gpghome.N1dLTrmHuI/S.dirmngr' failed: No such file or directory\ngpg: keyserver receive failed: No dirmngr\n", "stderr_lines": ["Warning: apt-key output should not be parsed (stdout is not a terminal)", "gpg: failed to start the dirmngr '/usr/bin/dirmngr': No such file or directory", "gpg: connecting dirmngr at '/tmp/apt-key-gpghome.N1dLTrmHuI/S.dirmngr' failed: No such file or directory", "gpg: keyserver receive failed: No dirmngr"], "stdout": "Executing: /tmp/apt-key-gpghome.N1dLTrmHuI/gpg.1.sh --keyserver ha.pool.sks-keyservers.net --recv XxXxX\n", "stdout_lines": ["Executing: /tmp/apt-key-gpghome.N1dLTrmHuI/gpg.1.sh --keyserver ha.pool.sks-keyservers.net --recv XxXxX"]}
```

**Comment by alessfg** [comment]
Hi @Am1rr3zA!

Few comments:
1. As you have already figure out - if you download the role from Ansible Galaxy you will need to use `nginxinc.nginx` instead of `ansible-role-nginx`. 
2. `nickdtodd.system-limits` is not a requirement for the NGINX Ansible role - perhaps it's a `datatog` or `gatling` dependency?
3. It looks as if the error you are getting has to do with missing the `dirmngr` package. Could you try installing it and trying to run the role again? If that doesn't work it'd be helpful if you could provide as much details as possible about your environment.

**Comment by Am1rr3zA** [comment]
so you saying ansible-role-nginx depends on dirmngr?

**Comment by alessfg** [comment]
In my testing I have found that to be the case if you are using Debian Stretch. Otherwise the role should install without any issues.

**Comment by alessfg** [comment]
Dependency fixed in #50
